@stellar/anchor-tests/src/tests/sep31/transactions.ts

```diff
@@ -117,7 +121,25 @@ const canCreateTransaction: Test = {
       ),
     };
     result.networkCalls.push(postTransactionsCall);
-    const responseBody = await makeRequest(
+
+    let assetCode = config.assetCode;
+    if (assetCode === undefined) {
+      throw "asset not configured";
+    }
+    quotesRequired = getQuotesRequiredFromInfo(
+      this.context.expects.sep31InfoObj,
+      assetCode,
+    );
+
+    // If quotes are required, ignore this test, this will be addressed in SEP 31+38 tests
+    if (quotesRequired) {
+      await makeRequest(postTransactionsCall, 400, result, "application/json");
+      this.context.provides.transactionId = null;
+      return result;
+    }
```

Comment:
So this is testing that if quotes are required, that the `POST /transactions` call fails with a 400? Maybe we should add a failure mode that describes this, that way its clear to the user why the test failed.

Reply:
I see what you're saying, but I think we also don't want this test to fail when it's being used for validation if a 400 is the expected behavior.